web/web.go

```diff
@@ -46,17 +47,20 @@ func staticHandler(handler http.Handler) http.Handler {
 	})
 }
 
-var browsersNotSupported string = "MSIE/8;MSIE/9;MSIE/10;Internet Explorer/8;Internet Explorer/9;Internet Explorer/10;Safari/7;Safari/8"
+//List should be of minimum required browser version. Do not place a ; at the end of this string.
+var browsersNotSupported string = "MSIE/11;Internet Explorer/11;Safari/9;Chrome/43;Edge/15;Firefox/52"
```

Comment:
Can we declare this as a slice instead of string splitting it?

Reply:
Maybe a slice of structs for the fields? Avoid all the strange parsing?

Reply:
@crspeller Yeah, I can look into it. I used what was already there, a slice makes more sense.